src/main/java/de/internetsicherheit/brl/bloxberg/cache/Launcher.java

```diff
@@ -28,27 +28,26 @@ public static void main(String[] args) throws IOException {
         parsedArgs[4] = "cli";
         for (int i = 0; i < args.length; i++) {
             String[] parts;
-            if (args[i].contains("--ui=")) {
+            if(args[i].contains("=")) {
```

Comment:
Actually, we should not add such complex code without having unit tests for it :slightly_smiling_face:

Reply:
hm ok will write tests for it. writing tests feels uncomfortable for me. hope that will change once i get used to it.

Reply:
I can understand this and this is normal. That's why I insist on it. Best way to learn and practice it is just writing them :slightly_smiling_face: